ocs_ci/ocs/utils.py

```diff
@@ -595,6 +595,31 @@ def create_oc_resource(
     occli.create(cfg_file)
 
 
+def get_pod_name(pattern='client', namespace='openshift-storage'):
```

Comment:
Instead of searching for pattern How about adding a label  in SmallFile.yaml and searching the pod with labels using ```ocs_ci/ocs/resources/pod.py:def get_pods_having_label(label, namespace)```.

For ex:
```
apiVersion: v1
kind: Pod
metadata:
  name: nginx
  labels:
    env: test
spec:
```
I see that 'kind' here is benchmark, I am guessing still label should be applicable.

Reply:
That was the method I had tried initially and I dont remember now it ends up creating 3 objects with same label (or it just created a label for one pod or something) and query returned immediately.

In anycase we still need *exact* individual pod name(in this case client) for later use

Reply:
Resolving!